src/custom/utils/appzi.ts

```diff
@@ -42,6 +42,10 @@ declare global {
 type AppziCustomSettings = {
   userTradedOrWaitedForLong?: true
   isTestNps?: true // to trigger test rather than prod NPS
+  secondsSinceOpen?: number // how long has this order been open (in seconds)?
+  waitedTooLong?: true
+  expired?: true
+  traded?: true
```

Comment:
Changed a little bit.

Now there's a flag for every type of trigger
And how long the order has been open/pending is tracked for all types in `secondsSinceOpen`